app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/address/ContactAddress.scala

```diff
@@ -99,20 +85,18 @@ object Address {
       )(identity)(Some(_))
     )
 
-  implicit class AddressOps(private val a: Address) extends AnyVal {
+  implicit class AddressOps(private val a: ContactAddress) extends AnyVal {
     def getAddressLines(implicit messages: Messages): List[String] = {
-      val lines = a match {
-        case u: UkAddress    => List(Some(u.line1), u.line2, u.town, u.county, Some(u.postcode.value))
-        case n: NonUkAddress =>
-          List(
-            Some(n.line1),
-            n.line2,
-            n.line3,
-            Some(n.line4),
-            Some(n.postcode),
-            messages.translate(s"country.${n.country.code}", Seq.empty)
-          )
-      }
+      val lines =
+        List(
+          Some(a.line1),
+          a.line2,
+          a.line3,
+          Some(a.line4),
+          Some(a.postcode),
+          messages.translate(s"country.${a.country.code}", Seq.empty)
+        )
+
       lines.collect { case Some(s) => s }
```

Comment:
```suggestion
        List(
          Some(a.line1),
          a.line2,
          a.line3,
          Some(a.line4),
          Some(a.postcode),
          messages.translate(s"country.${a.country.code}", Seq.empty)
        ).flatten
```

Reply:
Nice, @gabor-dorcsinecz can you make this change in your last ticket?